core/iwasm/libraries/libc-wasi/sandboxed-system-primitives/src/posix.c

```diff
@@ -647,7 +648,7 @@ fd_table_insert(wasm_exec_env_t exec_env, struct fd_table *ft,
         fd_object_release(exec_env, fo);
         return convert_errno(errno);
     }
-
+    
```

Comment:
Had better not add trailing spaces, or code guideline check CI will report error.